src/index.html

```diff
@@ -3,11 +3,396 @@
+              </a>
+              <p class="events__caption">
+                March 27, 2021 - October 30, 2022
+              </p>
+                
+             
+                
+            </section>
+
+            <section>
+              <div class="events__gallery3">
+                <div class="events__photo3">
+                  <img 
+                    src="images/sixth.png"
+                    class="events__image3" 
+                    alt="walt Disney"
+                  >
+  
+                </div>
+  
+                <div class="events__photo3">
+                  <img 
+                    src="images/seventh.png" 
+                    class="events__image3"
+                    alt="walt Disney"
+                  >
+                  
+                </div>
+  
+              </div>
+  
+              
+            
+              <a href="#" class="events__caption-link">
+                Inspiring Walt Disney: 
+                The Animation of French Decorative Arts
+              </a>
+              <p class="events__caption">
+                December 10, 2021 - March 6, 2022
+              </p>
+                
+                
+            </section>
+
+          </section>
+          
+        </div>
+
+        <button class="events__exhibition">View all exhibitions
+
+        </button>
+       
+
+        
+        <section class="explore">
+          <h2 class="explore__title">More to Explore Online</h2>
+          <div class="explore__conntainer">
+            <div class="explore__content">
+              <img src="images/explore1.png" alt="image" class="explore__image">
+              <a href="#" class="explore__link">Collection Areas</a>
+              <p class="explore__intro">These collection areas study, exhibit, and care for the objects in The Met collection.</p>
+            </div>
+  
+            <div class="explore__content">
+              <img src="images/explore2.png" alt="image" class="explore__image">
+              <a href="#" class="explore__link">Travel</a>
+              <p class="explore__intro">Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.</p>
+            </div>
+            <div class="explore__content">
+              <img src="images/explore3.png" alt="image" class="explore__image">
+              <a href="#" class="explore__link">Virtual group tours</a>
+              <p class="explore__intro">Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.</p>
+            </div>
+
+          </div>
+          
+
+        </section>
+
+        <section class="contact">
+          <div class="contact-container">
+
+            <h2 class="contact__title">Contact us</h2>
+
+            <h2 class="contact__name">The Met Fifth Avenue</h2>
+            <h2 class="contact__phone">Phone</h2>
+            <h2 class="contact__phoneNumber">+1 212-535-7710</h2>
```

Comment:
has to be an active link along with all other phones, addresses, emails
>5	When you click on phone icon or phone number in contacts section, make sure that there is no 404 error, make it a real link to start a call on device
>7	When clicking on any location / address - prevent errors and make it to open location in Google Maps
>9	Location-related addresses / links / images should open google maps in a new tab `target="_blank"`

plus
>4	Change text color on hover for phone, email and address